src/app/course-page/course-page.component.html

```diff
@@ -0,0 +1,21 @@
+<section>
+  <div class="row course__search">
+    <mat-form-field appearance="outline" color="primary">
+      <mat-label>Course</mat-label>
+      <mat-icon matPrefix>search</mat-icon>
+      <input matInput placeholder="Text to search" [(ngModel)]="courseInput" />
+    </mat-form-field>
+    <button class="search__button" (click)="searchCourse(courseInput)">Search</button>
+  </div>
+
+  <div class="row course__add">
+    <button (click)="addCourse()">
```

Comment:
Doesn't make sense to wrap it in a button. Can be just click on icon.

Reply:
Leave it in button. You won't have to add CSS for the pointer additionally it is better for accessibility to have it in button.

Reply:
Sorry, missed 'Add Course' text as well. Thought it is just icon.